canonical-web-and-design/ubuntu.com
# Update k8s datasheet

## Done

- Uploaded the datasheet to the asset server
- Put a note on the old datasheet tag info pointing people to the new one
- Updated the copy docs
- Deleted the old /support/plans-and-pricing page
- Updated the three pages

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- http://0.0.0.0:8001/ai/consulting - [copy doc](https://docs.google.com/document/d/19Mx1zRJKqH2hz5kRF_oDsgyFUw67FgJvW4I_OGda-wk/edit#)
- http://0.0.0.0:8001/kubernetes - [copy doc](https://docs.google.com/document/d/1G-h1eWiBSKqG9oWB15Pup8VDe1axBUwQGvtkwJ9g8T0/edit#heading=h.xnpkx4ypduk)
- http://0.0.0.0:8001/kubernetes/managed - [copy doc](https://docs.google.com/document/d/15jzhasokjH5Ql9Za56ZxRknXNNhEXYdIFXXtLODGxns/edit)

## Issue / Card

Fixes #5108
